biome.json

```diff
@@ -34,6 +34,11 @@
       "enabled": false
     }
   },
+  "css": {
+    "formatter": {
+      "enabled": false
+    }
+  },
```

Comment:
we use prettier instead

Reply:
We could also sync config in `.vscode/settings.json` to instruct for choosing right formatter for VS Code.

Reply:
done by modifying `.vscode/settings.json`